test/TEST-16-ISSUE-518/test.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/python
+
+#
+#  Copyright 2017 Michal Sekletar <msekleta@redhat.com>
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+#
+#  systemd is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public License
+#  along with python-systemd; If not, see <http://www.gnu.org/licenses/>.
+
+import unittest
+import time
+import os
+import tempfile
+
+from pydbus import SystemBus
+from enum import Enum
```

Comment:
Hm, right; I wrote that because there is no Python 3 in Centos 7 (where we run the "default" test), but there we just run the complete build without Python support:

    checking for a Python interpreter with version >= 3... none

So I guess py 3 only is fine.